app/models/assets_creator.rb

```diff
@@ -1,9 +1,73 @@
 class AssetsCreator < ApplicationRecord
-  belongs_to :asset, :polymorphic => true
-  belongs_to :creator, :class_name => 'Person'
+  belongs_to :asset, polymorphic: true
+  belongs_to :creator, class_name: 'Person', optional: true
+  alias_method :person, :creator
 
   include Seek::Rdf::ReactToAssociatedChange
+  include Seek::OrcidSupport
+  include Seek::BioSchema::Support
+
   update_rdf_on_change :asset
 
-  default_scope { order(:id) }
+  default_scope { order(:pos) }
+
+  def family_name
+    creator ? creator.last_name : super
+  end
+  alias_method :last_name, :family_name
```

Comment:
looks like we need some refactoring for consistent use of family/given name over first and last. Maybe not for this PR but worth adding an issue